tests/viz_tests.py

```diff
@@ -1163,3 +1163,26 @@ def test_process_data_resample(self):
             .tolist(),
             [1.0, 2.0, np.nan, np.nan, 5.0, np.nan, 7.0],
         )
+
+    def test_apply_roling(self):
+        datasource = self.get_datasource_mock()
+        df = pd.DataFrame(
+            index=pd.to_datetime(
+                ["2019-01-01", "2019-01-02", "2019-01-05", "2019-01-07"]
+            ),
+            data={"y": [1.0, 1.0, 1.0, 1.0]},
+        )
+        self.assertEqual(
+            viz.BigNumberViz(
+                datasource,
+                {
+                    "metrics": ["y"],
+                    "rolling_type": "cumsum",
```

Comment:
It would be good to test the other `rolling_type` values - this is only testing one branch of a nested `if` statement.

Reply:
Agreed, that was the plan to fast follow once I manage to get `tox` working locally. Somehow it's really hard to install python3.6 on a new Mac.